sdks/python/apache_beam/coders/row_coder.py

```diff
@@ -85,6 +87,13 @@ def as_deterministic_coder(self, step_label, error_message=None):
   def to_type_hint(self):
     return self._type_hint
 
+  def as_cloud_object(self, coders_context=None):
```

Comment:
Aren't cloud objects a pre-portability concept that will be deprecated?

Reply:
I agree they are pre-portability, I don't know where we are on them being deprecated long term. My understanding is that for portability their generation is moved from the Beam SDK to the Dataflow Service.

This PR is bringing Python up to feature parity with Java. My understanding is that there is no support for Schema metadata in portability yet, but I'll be looking into that next.